vendor/vtd/conanfile.py

```diff
@@ -0,0 +1,62 @@
+from pathlib import Path
+import shutil
+
+from conans import CMake, ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+
+
+class VtdConan(ConanFile):
+    name = "vtd"
+    version = "2.2.0"
+    license = "Proprietary"
+    url = "https://vires.mscsoftware.com"
+    no_copy_source = True
+    description = "Include binary files in C/C++"
+    topics = ("simulator", "vires")
+    settings = "os", "arch"
+    options = {
+        "with_osi": [True, False],
+        "with_road_designer": [True, False],
+    }
+    default_options = {
+        "with_osi": True,
+        "with_road_designer": False,
+    }
+
+    _archive_base = "vtd.2.2.0.Base.20181231.tgz"
+    _archive_osi = "vtd.2.2.0.addOns.OSI.20190513.tgz"
+    _archive_rod = "vtd.2.2.0.addOns.ROD.Base.20190313.tgz"
+    exports_sources = [
+        _archive_base,
+        _archive_osi,
+        _archive_rod,
+    ]
+
+    _root_dir = "VTD.2.2"
+
+    def configure(self):
+        if self.settings.os == "Windows":
+            raise ConanInvalidConfiguration("VTD binaries do not exist for Windows")
+
+    def build(self):
+        src = Path(self.source_folder)
+        dst = Path(self.build_folder)
+
+        def extract_archive(archive):
+            print("Extracting: {}".format(archive))
+            tools.untargz(src / archive, dst)
```

Comment:
With conan v1.32.1, I am getting errors related to the `untargz` arguments:
```
while calling 'extract_archive', line 47
	tools.untargz("{}".format(src / archive), dst)
	TypeError: join() argument must be str or bytes, not 'PosixPath'
```
This works for me:
`tools.untargz("{}".format(src / archive), "{}".format(dst))`

Reply:
I guess this explains why it works for me.

Reply:
After fixing my conan installation and configuration, I am also using Python 3.6 and it works as you said.